core/templates/pages/diagnostic-test-player-page/diagnostic-test-player.component.html

```diff
@@ -0,0 +1,70 @@
+<div>
+  <button class="btn oppia-diagnostic-test-button oppia-back-button"
+          (click)="returnBackToClassroom()">
+    BACK
+  </button>
+</div>
+<div class="oppia-diagnostic-test-session">Learner Diagnostic Test</div>
+<mat-card class="introductory-card">
+  <div class="oppia-introductory-container">
+    <div class="oppia-image">
+      <img [src]="OPPIA_AVATAR_IMAGE_URL" alt="">
+    </div>
+    <div class="test-introduction-container">
+      <p class="test-introduction-text">
+        This test will help us recommend a topic for you.
+      </p>
+      <p class="test-introduction-text">
+        Please note that you will not be able to return to previous questions
+        after moving to the next question.
+      </p>
+    </div>
+  </div>
+  <button class="btn oppia-diagnostic-test-button start-test-button"
+          disabled="true">
+    START THE TEST
```

Comment:
This and all the other text on this page needs to be internationalized.

Also, in general, for all-caps things -- write it normally (e.g. "Start the Test") and use CSS styles to display it as all-caps.

Reply:
Updated